src/components/hocs/withFullscreen.js

```diff
@@ -3,20 +3,16 @@ import Flex from "@netdata/netdata-ui/lib/components/templates/flex"
 import Layer from "@netdata/netdata-ui/lib/components/templates/layer"
 import { useAttributeValue } from "@/components/provider"
 
-export default Component => {
```

Comment:
Why is this change here?

Reply:
It's irrelevant to this PR!
Due to [this bug](https://github.com/netdata/cloud-frontend/issues/3193) that I encountered while doing the update, I opened this file (among others) to do a tiny research for its root and my IDE's linter was giving me a "warning" for redundant const `FullscreenComponent`.
If you want me to change it back please say so! 😄

Reply:
Kept it as it was previously! 😄